src/main/java/com/woowacourse/pelotonbackend/member/presentation/dto/MemberCashUpdateRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.woowacourse.pelotonbackend.member.presentation.dto;
+
+import java.beans.ConstructorProperties;
+
+import javax.validation.constraints.NotNull;
+
+import com.woowacourse.pelotonbackend.vo.Cash;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+@AllArgsConstructor(onConstructor_ = {@ConstructorProperties({"cash"})})
```

Comment:
```suggestion
@AllArgsConstructor(onConstructor_ = @ConstructorProperties({"cash"}))
```
값이 하나만 들어가니까 {}는 빼도 될 것 같아요! 다른 클래스들도 바꿔주세요